src/main/java/com/googleinterns/zoomtube/servlets/TranscriptServlet.java

```diff
@@ -157,17 +159,21 @@ private void writeResponseAsJson(HttpServletResponse response, ImmutableList<Tra
   private Entity createLineEntity(Node node, long lectureId) {
     Element element = (Element) node;
     String lineContent = node.getTextContent();
-    String lineStart = element.getAttribute(ATTR_START);
-    String lineDuration = element.getAttribute(ATTR_DURATION);
-
+    Float lineStart = Float.parseFloat(element.getAttribute(ATTR_START));
+    Float lineDuration = Float.parseFloat(element.getAttribute(ATTR_DURATION));
```

Comment:
nit: I would declare these right before they are used.

Reply:
I kinda like the distinction between getting and setting the attributes. Is that unadvised or are you referring to something else?

Reply:
This is a nit. I don't feel strong either way.